content/webapp/pages/stories.js

```diff
@@ -223,13 +225,14 @@ export class StoriesPage extends Component<Props> {
         </SpacingSection>
 
         <SpacingSection>
-          <SectionHeader
-            title="You may have missed"
-            linkText="More articles"
-            linkUrl="/articles"
-          />
-
-          <CardGrid items={articles.slice(5, 11)} />
+          <SpacingComponent>
+            <SectionHeader title="You may have missed" />
+          </SpacingComponent>
+          <SpacingComponent>
+            <CardGrid items={articles.slice(5, 11)} itemsPerRow={3}>
+              <MoreLink name={`More articles`} url={`/articles`} />
```

Comment:
Ah, okay, I can see that this might be part of the component - could we type this, or potentially just send the props through and create the link in the `CardGrid` component, which would aim for more consistency by not just lumping any child nodes in this?

Reply:
Yeah, I wondered. Link prop probably safest. Will update